participant/helpers.py

```diff
@@ -0,0 +1,11 @@
+from .models import Participant
+from exercise.models import Trial
+
+
+def create_participant(exercise, trial=None):
```

Comment:
I'd rename the parameter `trial` to `trial_id` to be super clear what this function expects.

Reply:
Good catch. Will change it.